Poker.BE/Poker.BE.Domain/Security/UserManager.cs

```diff
@@ -44,22 +50,28 @@ public class UserManager
             return false;
```

Comment:
# 👎 Magic number
`password.Length >= 6` - **6** is a magic number here. please use:
```c#
#region Constants
    public const int MINIMUM_PASSWORD_LENGTH = 6;
#endregion
```

this applies to all magic numbers you have in your code - use constants definitions instead. the constants region should be the first region at the class.